### DISCUSSION TITLE
XRDP Remote App Support?

### DISCUSSION THREAD

**Discussion by William-Proctor** [neutrinolabs/xrdp#3020] [Q&A] [answered]
Hi All,

Has any progress been made on a remote app, I'd like to use it if possible.

I know some work was done by [aquesnel](https://github.com/aquesnel) but I haven't seen any update about his modifications from here https://github.com/neutrinolabs/xrdp/issues/1038 and the issue was closed awaiting feedback.  I've also seen that in June last year, it was asked if it was supported and the answer was no at the time. (This is also Closed.)

If there is work to be done could you highlight the areas for a remote app to function correctly? (So I can take a stab at it.)

Thanks in advance, 
Will

**Comment by matt335672** [comment, marked as answer]
No, there's no more progress in this area. We've been working on GFX support for a bit, and at the moment other activities are seen as higher priority by the core developers.
